bitnami/flux/Chart.yaml

```
@@ -43,4 +43,4 @@ maintainers:
 name: flux
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/flux
-version: 2.1.3
+version: 2.1.4
```

Just for consistency with other similar PRs
```suggestion
version: 2.2.0
```